examples/tutorials/get-started/flow-as-function.ipynb

```diff
@@ -39,6 +39,24 @@
     "You will need to have a connection named \"new_ai_connection\" to run flow with new connection."
    ]
   },
+  {
```

Comment:
Add some starting words for this sample, e.g.
![image](https://github.com/microsoft/promptflow/assets/47586720/520aa1cf-3cab-4362-94eb-7bba505d222d)